extensions/vscode/src/events.ts

```diff
@@ -177,15 +177,24 @@ export class EventStream extends Readable implements Disposable {
   }
 
   private processMessage(msg: EventStreamMessage) {
-    const localId = msg.data.localId;
-    if (localId && this.canceledLocalIDs.includes(localId)) {
+    // Some log messages passed on from Connect include
+    // the localId using snake_case, rather than pascalCase.
+    // To filter correctly, we need to check for both.
+
+    const localIdForm1 = msg.data.localId;
+    if (localIdForm1 && this.canceledLocalIDs.includes(localIdForm1)) {
+      // suppress and ignore
+      return;
+    }
+    const localIdForm2 = msg.data.local_id;
+    if (localIdForm2 && this.canceledLocalIDs.includes(localIdForm2)) {
       // suppress and ignore
       return;
     }
-    // Trace message
-    // console.debug(
-    //   `eventSource trace: ${event.type}: ${JSON.stringify(event)}`,
-    // );
+
+    // // Trace message
+    // console.debug(`eventSource trace: ${msg.type}: ${JSON.stringify(msg)}`);
+
```

Comment:
Should we just remove this since it is commented out?

Reply:
I've left it in, because I come back and need full message tracing every month or so. Let me add a comment that indicates to un-comment this if full message tracing is needed.

Reply:
Oops looks like your change left the debug uncommented 😅

Reply:
Great catch. Fixed!